lib/tasks/temporary/env_variables.rake

```diff
@@ -0,0 +1,17 @@
+# Create a .env file from your application.yml file
+task create_dot_env_file: :environment do
+  env_file = File.open(".env", "w")
+
+  File.open("config/application.yml", 'r') do |file|
+    file.each_line do |line|
```

Comment:
**suggestion (non-blocking):** Instead of `open` + `each_line` you can also use `foreach`:

```ruby
File.foreach("config/application.yml", 'r') do |line|
  new_line = line.gsub(": ", "=")
  ...
end
```

You can use `open` for `env_file` though:

```ruby
File.open(".env", "w") do |env_file|
  File.foreach("config/application.yml", 'r') do |line|
    new_line = line.gsub(": ", "=")
    ...
  end
end
```

Reply:
`File.foreach` here does not split it up by line, it kinda of haphazardly splits it up so when I tried it I got exports in the middle of lines everywhere. I am going to stick with my inner block but then use a block to create the new env file like you suggest